src/Microsoft.AspNetCore.Razor.VSCode/src/RazorDocumentFactory.ts

```diff
@@ -27,17 +27,17 @@ export function createDocument(uri: vscode.Uri) {
 }
 
 function createProjectedHtmlDocument(hostDocumentUri: vscode.Uri) {
-    // Index.cshtml => __Index.cshtml.html
-    const projectedPath = `__${hostDocumentUri.path}.html`;
+    // Index.cshtml => Index.cshtml__virtual.html
+    const projectedPath = `${hostDocumentUri.path}__virtual.html`;
```

Comment:
TBH I haven't fully understood the scenarios where this path shows up anywhere that the change would make a difference. Can the user ever see the `*__virtual.html` filenames anywhere?

Reply:
No they cannot, this was a fix I added after debugging through vscode itself. In their internal representation of our files they strip the `__` and have their URI authority as `__`. Therefore, when they try and cross-lookup our files they request funky paths with `__` at the beginning of the rooted path.